src/SFA.DAS.ApprenticeFeedback.Application/Commands/CreateApprenticeFeedback/CreateApprenticeFeedbackHandler.cs

```diff
@@ -47,15 +47,9 @@ public async Task<CreateApprenticeFeedbackResponse> Handle(CreateApprenticeFeedb
             var feedback = new Domain.Entities.ApprenticeFeedbackResult
             {
                 ApprenticeFeedbackTargetFeedbackId = apprenticeFeedbackTarget.Id,
-                Ukprn = request.Ukprn,
-                LarsCode = request.LarsCode,
                 StandardUId = request.StandardUId,
-                StandardReference = request.StandardReference,
                 DateTimeCompleted = DateTime.UtcNow,
                 ProviderRating = request.OverallRating.ToString(),
-                //ProviderName = request.ProviderName,
```

Comment:
We've lost creating the provider attributes, these won't be saved anymore. Did you try running it via Postman to confirm everything saves?

Reply:
I might need some help with postman (faking some of the data)